Seg fault on empty command in included Taskfile

- Task version: 3.9.0
- Operating System: Ubuntu 18.04

The issue described in https://github.com/go-task/task/issues/534 is resolved when there is an empty command in the Taskfile

```yaml
version: '3'

tasks:
  hello:
    desc: 'hi'
    cmds:
      - echo "hi"
      -
```

This now works perfectly.


It is not resolved, however, when the empty command is in an included Taskfile:

`Taskfile.yml`
```yaml
version: '3'

includes:
  add: add.Taskfile.yml
```

`add.Taskfile.yml`
```yaml
version: '3'

tasks:
  hello:
    desc: 'hi'
    cmds:
      - echo "hi"
      - 
```

`task -l` fails with:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x56d697]

goroutine 1 [running]:
github.com/go-task/task/v3/taskfile.Merge(0xc000124000, 0xc0001241b0, {0xc0000e9798, 0x1, 0x1})
        /home/runner/go/pkg/mod/github.com/go-task/task/v3@v3.9.0/taskfile/merge.go:52 +0x497
github.com/go-task/task/v3/taskfile/read.Taskfile.func1({0xc0000aac68, 0x3}, {{0xc0000aaf10, 0x10}, {0x0, 0x0}, 0x0, 0x0})
        /home/runner/go/pkg/mod/github.com/go-task/task/v3@v3.9.0/taskfile/read/taskfile.go:103 +0x4ad
github.com/go-task/task/v3/taskfile.(*IncludedTaskfiles).Range(0xc0000be3e0, 0xc0000e9bc0)
        /home/runner/go/pkg/mod/github.com/go-task/task/v3@v3.9.0/taskfile/included_taskfile.go:78 +0x124
github.com/go-task/task/v3/taskfile/read.Taskfile({0x0, 0x0}, {0x6a8208, 0x10})
        /home/runner/go/pkg/mod/github.com/go-task/task/v3@v3.9.0/taskfile/read/taskfile.go:40 +0x15c
github.com/go-task/task/v3.(*Executor).Setup(0xc0000a72b0)
        /home/runner/go/pkg/mod/github.com/go-task/task/v3@v3.9.0/task.go:113 +0x7b
main.main()
        /home/runner/go/pkg/mod/github.com/go-task/task/v3@v3.9.0/cmd/task/task.go:147 +0x8d9
```

--- a/task_test.go
+++ b/task_test.go
@@ -756,14 +756,14 @@ func TestIncludesCallingRoot(t *testing.T) {
 }
 
 func TestIncludesOptional(t *testing.T) {
-  tt := fileContentTest{
-    Dir:       "testdata/includes_optional",
-    Target:    "default",
-	  TrimSpace: true,
-	  Files: map[string]string{
-		  "called_dep.txt": "called_dep",
-	  }}
-  tt.Run(t)
+	tt := fileContentTest{
+		Dir:       "testdata/includes_optional",
+		Target:    "default",
+		TrimSpace: true,
+		Files: map[string]string{
+			"called_dep.txt": "called_dep",
+		}}
+	tt.Run(t)
 }
 
 func TestIncludesOptionalImplicitFalse(t *testing.T) {
@@ -1033,6 +1033,7 @@ func TestIgnoreNilElements(t *testing.T) {
 	}{
 		{"nil cmd", "testdata/ignore_nil_elements/cmds"},
 		{"nil dep", "testdata/ignore_nil_elements/deps"},
+		{"nil include", "testdata/ignore_nil_elements/includes"},
 		{"nil precondition", "testdata/ignore_nil_elements/preconditions"},
 	}
 

--- a/taskfile/merge.go
+++ b/taskfile/merge.go
@@ -49,7 +49,7 @@ func Merge(t1, t2 *Taskfile, namespaces ...string) error {
 			dep.Task = taskNameWithNamespace(dep.Task, namespaces...)
 		}
 		for _, cmd := range v.Cmds {
-			if cmd.Task != "" {
+			if cmd != nil && cmd.Task != "" {
 				cmd.Task = taskNameWithNamespace(cmd.Task, namespaces...)
 			}
 		}

--- a/testdata/ignore_nil_elements/includes/Taskfile.yml
+++ b/testdata/ignore_nil_elements/includes/Taskfile.yml
@@ -1,0 +1,9 @@
+version: '3'
+
+includes:
+  inc: inc.yml
+
+tasks:
+  default:
+    cmds:
+      - task: inc:default

--- a/testdata/ignore_nil_elements/includes/inc.yml
+++ b/testdata/ignore_nil_elements/includes/inc.yml
@@ -1,0 +1,7 @@
+version: '3'
+
+tasks:
+  default:
+    cmds:
+      - 
+      - echo "string-slice-1"

